tox.ini

```diff
@@ -1,22 +1,64 @@
 [tox]
 envlist =
-    py27-dj111{,-grp}
-    py{36,37}-dj{111,20}{,-grp}
+    py3{6,7,8,9}-django{22,30,31,32}
+
+[pytest]
+django_find_project = false
+DJANGO_SETTINGS_MODULE=select2.tests.settings
 
 [testenv]
-commands =
-    python runtests.py {posargs} --noinput
+description = Run tests in {envname} environment
 setenv =
-    DJANGO_SELENIUM_TESTS = 1
-passenv =
-    TRAVIS
-    TRAVIS_*
-    DATABASE_URL
+    PYTHONPATH = {toxinidir}:{toxinidir}/tests:{env:PYTHONPATH:}
+commands = {posargs:pytest}
 deps =
+    pytest>=5.2.0
+    pytest-django
     selenium
     django-selenosis
-    dj-database-url
-    dj111: Django~=1.11.0
-    dj20: Django~=2.0.0
-    dj111-grp: django-grappelli==2.10.2
-    dj20-grp: django-grappelli==2.10.2
+    django22: Django>=2.2,<3.0
+    django31: Django>=3.1,<3.2
+    django32: Django>=3.2b1,<4.0
+    django22-grp: django-grappelli==2.13.4
+    django31-grp: django-grappelli==2.14.4
+    django32-grp: django-grappelli==2.15.1
+
+[testenv:clean]
+description = Clean all build and test artifacts
+skipsdist = true
+skip_install = true
+deps =
+whitelist_externals =
+    find
+    rm
+commands =
+    find {toxinidir} -type f -name "*.pyc" -delete
+    find {toxinidir} -type d -name "__pycache__" -delete
+    rm -f {toxinidir}/tests/db.sqlite {toxworkdir} {toxinidir}/.pytest_cache {toxinidir}/build
+
+[testenv:pep8]
+description = Run PEP8 flake8 against the select2/ package directory
+skipsdist = true
+skip_install = true
+basepython = python3.7
```

Comment:
Should this be 3.6?

Reply:
Let's default to 3.7. Elsewhere we have removed 3.6, so we can do so here as well.